rpcs3/Emu/Io/usio.cpp

```diff
@@ -190,75 +190,69 @@ void usb_device_usio::translate_input_taiko()
 	constexpr le_t<u16> c_hit = 0x1800;
 	le_t<u16> digital_input = 0;
 
-	auto translate_from_pad = [&](usz pad_number, usz player)
+	const auto translate_from_pad = [&](usz pad_number, usz player)
 	{
-		if (!is_input_allowed())
-		{
-			return;
-		}
-
-		const auto& pad = ::at32(handler->GetPads(), pad_number);
-		if (!(pad->m_port_status & CELL_PAD_STATUS_CONNECTED))
-		{
-			return;
-		}
-
 		const usz offset = player * 8ULL;
 		auto& status = m_io_status[0];
 
-		const auto& cfg = ::at32(g_cfg_usio.players, pad_number);
-		cfg->handle_input(pad, false, [&](usio_btn btn, u16 /*value*/, bool pressed)
-			{
-				switch (btn)
+		if (const auto& pad = ::at32(handler->GetPads(), pad_number); (pad->m_port_status & CELL_PAD_STATUS_CONNECTED) && is_input_allowed())
```

Comment:
I don't see any reason at all to indent all of this.
Just make the function a lambda and pass it to handle_input.
Then the code will be much easier to review.

Reply:
Fixed.
I feel sorry about Visual Studio's auto indent.